controllers/provider-azure/pkg/azure/types.go

```diff
@@ -46,6 +47,16 @@ const (
 	StorageAccount = "storage-account"
 	// StorageKey is a constant for the key in a cloud provider secret and backup secret that holds the Azure secret storage access key.
 	StorageKey = "storage-key"
+	//TOREMOVE: To keep up with naming convention we will change these constant related to storage. But will removed at the time we completely move to
+	// shared bucket approach.
+
+	// NewStorageAccount is a constant for the key in a cloud provider secret and backup secret that holds the Azure account name.
+	NewStorageAccount = "storageAccount"
+	// NewStorageKey is a constant for the key in a cloud provider secret and backup secret that holds the Azure secret storage access key.
+	NewStorageKey = "storageKey"
+
+	// AzureBlobStorageHostName is the host name for azure blob storage service.
+	AzureBlobStorageHostName = "blob.core.windows.net"
```

Comment:
Can't this be imported from the Azure library somewhere?

Reply:
Unfortunately not. I looked for it. I couldn't find it, even i was surprised to not see the constant defined for this. If you find it, i'll be happy to use it.